### DISCUSSION TITLE
How to Reinstall MySQL in Laragon

### DISCUSSION THREAD

**Discussion by falehjamal** [leokhoa/laragon#902] [Q&A]
![image](https://github.com/user-attachments/assets/545d26ab-873d-498a-9465-2d1ecbf6c36f)

Hello, good evening. I would like to ask how to reinstall MySQL in Laragon, just MySQL only. I had tried to modify it before, but it ended up messing up my MySQL setup. I want to reset my MySQL database in Laragon from scratch. Thank you.

**Comment by FlavioSuarez** [comment]
Hi,

[Here](https://gist.github.com/meorajrul/b57803bf1b4ddfd2f93e6ad37c3ac5f2?permalink_comment_id=5088459#gistcomment-5088459) there is a step by step solution...

**Comment by ppfoong** [comment]
If you want to start from scratch, make sure MySQL is stopped, then rename your data directory under laragon to data-bak and create an empty data directory.

Start MySQL, the default files will be generated in the new data directory.
